service/src/main/kotlin/fi/espoo/evaka/invoicing/controller/FeeDecisionController.kt

```diff
@@ -107,7 +110,12 @@ class FeeDecisionController(
         @Suppress("DEPRECATION")
         user.requireOneOfRoles(UserRole.FINANCE_ADMIN)
         db.transaction { tx ->
-            val confirmedDecisions = service.confirmDrafts(tx, user, feeDecisionIds, Instant.now())
+            val confirmedDecisions = service.confirmDrafts(
+                tx,
+                user,
+                feeDecisionIds,
+                Instant.now().plus(Period.ofDays(env.nrOfDaysFeeDecisionCanBeSentInAdvance))
```

Comment:
Latest convention to using current time/date is to use `EvakaClock` and its `today()` and `now()` methods to make it easier to mock the current time in tests. So in practice you can add `evakaClock: EvakaClock` to the endpoints parameters.